spec/models/miq_expression/model_details_spec.rb

```diff
@@ -82,6 +82,10 @@
         result = described_class.model_details("Service", :typ => "all", :include_model => false, :include_tags => true)
         expect(result.map(&:first)).to include("My Company Tags : Auto Approve - Max CPU")
       end
+
+      it "Supports classes derived form ActsAsArModel" do
+        described_class.model_details("Chargeback", :typ => "all", :include_model => false, :include_tags => true)
```

Comment:
LOL I had mention to @gtanzillo that wrapping a test in `expect {}.to_not raise_error` kind of doesn't make sense, since if it blows up it will fail anyway, but yeah, it looks weird with no expectation.  What would probably be best is a test that actually verifies the output from the model_details.